caps/repo_test.go

```diff
@@ -0,0 +1,154 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2015 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package caps
+
+import (
+	"testing"
+
+	. "gopkg.in/check.v1"
+
+	"github.com/ubuntu-core/snappy/testutil"
+)
+
+func TestRepository(t *testing.T) {
+	TestingT(t)
+}
+
+type RepositorySuite struct {
+	// Typical repository with known types
+	repo *Repository
+	// Empty repository
+	emptyRepo *Repository
+}
+
+var _ = Suite(&RepositorySuite{})
+
+func (s *RepositorySuite) SetUpTest(c *C) {
+	s.repo = NewRepository()
+	s.emptyRepo = NewRepository()
+	err := LoadBuiltInTypes(s.repo)
```

Comment:
Because otherwise I'd have to add the File type each time and that's not what they are actually testing. I wanted to avoid having to continuously add the FileType over and over again.